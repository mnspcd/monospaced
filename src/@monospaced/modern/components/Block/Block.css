.mdrn .Block {
  align-items: center;
  background-color: var(--color-pink-1);
  border: calc(1rem / 16) dashed var(--color-pink-6);
  color: var(--color-pink-6);
  display: flex;
  font-family: var(--font-monospaced);
  font-variant: slashed-zero;
  font-weight: 300;
  height: 100%;
  justify-content: center;
  padding: 1.5em 0.75em;
  width: 100%;
}
