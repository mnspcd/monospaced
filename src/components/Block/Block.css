.Block {
  align-items: center;
  background-color: var(--mnspcd-color-pink-1);
  border: calc(1rem / 16) dashed var(--mnspcd-color-pink-6);
  color: var(--mnspcd-color-pink-6);
  display: flex;
  font-family: var(--font-monospaced);
  font-variant: slashed-zero;
  font-weight: 300;
  height: 100%;
  justify-content: center;
  padding: 1.5em 0.75em;
  width: 100%;
}
