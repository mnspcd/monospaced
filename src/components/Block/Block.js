import PropTypes from "prop-types";
import React from "react";

import "./Block.css";

const Block = ({ children, height, width }) => (
  <div className="Block" style={{ height, width }}>
    {children}
  </div>
);

Block.propTypes = {
  children: PropTypes.node,
  height: PropTypes.string,
  width: PropTypes.string,
};

export default Block;
