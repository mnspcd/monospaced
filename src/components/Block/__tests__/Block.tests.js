import React from "react";
import { render } from "enzyme";

import Block from "../Block";

describe("Block component", () => {
  it("should render correctly", () => {
    const component = render(<Block>children</Block>);
    expect(component).toMatchSnapshot();
  });
});
