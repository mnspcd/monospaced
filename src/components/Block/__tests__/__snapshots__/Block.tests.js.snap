// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Block component should render correctly 1`] = `
<div
  class="Block"
>
  children
</div>
`;
